core/backend/src/IModelDb.ts

```diff
@@ -2983,7 +2983,7 @@ export class BriefcaseDb extends IModelDb {
    * Execute the supplied function.
    * If the briefcase was read-only, reopen the native briefcase as read-only.
    * @note this._openMode is not changed from its initial value.
-   * @internal Exported strictly for tests.
+   * @internal Exported strictly for tests
```

Comment:
`@internal` APIs [by definition](https://www.itwinjs.org/learning/api-support-policies/) are **not** to be used outside of itwinjs-core repository. They provide no stability guarantees.